src/Featurizers/Components/DocumentStatisticsEstimator.h

```diff
@@ -138,13 +131,28 @@ class DocumentStatisticsTrainingOnlyPolicy {
     // ----------------------------------------------------------------------
     static constexpr char const * const     NameValue = DocumentStatisticsEstimatorName;
 
+    enum AnalyzerMethod {
```

Comment:
>AnalyzerMethod [](start = 9, length = 14)

This should be represented as a custom_enum in Featurizers.yaml

Reply:
will make an update tomorrow